src/scss/mixins/_labels.scss

```diff
@@ -16,18 +15,31 @@
 
 /// Additional info
 @mixin oFormsAdditionalInfo {
-	@include oTypographySans(0);
+	@include oColorsFor(o-forms-field-additional-info body, text);
 	display: block;
-	color: oColorsGetColorFor(o-forms-field-additional-info body, text);
+	font-size: $_o-forms-font-size-small;
```

Comment:
Same, why the change?

Reply:
We wanted to change the font size here. This has been updated to use `oTypographySize` based on our previous conversation.